cmd/restic/main.go

```diff
@@ -74,14 +74,13 @@ func (cmd CmdInit) Execute(args []string) error {
 	}
 
 	s := server.NewServer(be)
-
-	_, err = server.CreateKey(s, pw)
+	err = s.CreateMasterKey(pw)
```

Comment:
I think you should split the two functions (see [above](https://github.com/restic/restic/pull/166/files#r29563676)) and just add a call to `CreateConfig(key)` here.